zerver/webhooks/alertmanager/view.py

```diff
@@ -0,0 +1,62 @@
+# Webhooks for external integrations.
+from typing import Any, Dict, List
+
+from django.http import HttpRequest, HttpResponse
+
+from zerver.decorator import api_key_only_webhook_view
+from zerver.lib.request import REQ, has_request_variables
+from zerver.lib.response import json_success
+from zerver.lib.webhooks.common import check_send_webhook_message
+from zerver.models import UserProfile
+
+@api_key_only_webhook_view('AlertManager')
+@has_request_variables
+def api_alertmanager_webhook(request: HttpRequest, user_profile: UserProfile,
+                             payload: Dict[str, Any] = REQ(argument_type='body')) -> HttpResponse:
+    name_field = request.GET.get("name", "instance")
+    desc_field = request.GET.get("desc", "alertname")
+    topics = {}  # type: Dict[str, Dict[str, List[str]]]
+
+    for alert in payload["alerts"]:
+        labels = alert.get("labels", {})
+        annotations = alert.get("annotations", {})
+
+        name = labels.get(
+            name_field, annotations.get(name_field, "(unknown)"))
+        desc = labels.get(
+            desc_field, annotations.get(desc_field, "<missing field: {}>".format(desc_field)))
+
+        url = alert.get("generatorURL").replace("tab=1", "tab=0")
+
+        body = "{description} [:chart:]({url})".format(description=desc, url=url)
```

Comment:
It looks like this formatting is a bit different from that in the screenshot; can you update the screenshot?

Reply:
I will, though this reminds me: emoji don't appear to work inside link text:

![image](https://user-images.githubusercontent.com/9830/74408292-421ee700-4df1-11ea-9967-6d3fb74a1917.png)

(I wanted to link with the chart emoji, but that didn't work!)

Reply:
Yeah, I was going to say I haven't seen anyone do that before and am not sure whether it would work.  Open a separate issue?  I'd be curious for someone to spend time debugging why that doesn't work; it seems like something that'd be nice to allow.

Reply:
I've reverted the text to `(graph)`, which matches the screenshot again. Once emoji in links are in place, the chart emoji might be nice. :)